packages/common/client-utils/src/bufferBrowser.ts

```diff
@@ -100,23 +100,33 @@ export class IsoBuffer extends Uint8Array {
 	}
 
 	/**
+	 * Static constructor
 	 * @param value - (string | ArrayBuffer)
 	 * @param encodingOrOffset - (string | number)
 	 * @param length - (number)
+	 *
+	 * @privateRemarks TODO: Use actual types
 	 */
-	static from(value, encodingOrOffset?, length?): IsoBuffer {
+	// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any
+	static from(value: any, encodingOrOffset?: any, length?: any): IsoBuffer {
```

Comment:
Why not `unknown` here?

Reply:
It actually causes tsc to interpret the "is object" case differently, which causes build failures. The casting would have been pretty gross, so I opted to leave this be (at least for now).